### ISSUE TITLE
Please support Enter shortcut, Experience using it like a web page

### ISSUE THREAD

**Issue by fwnan** [LiLittleCat/intellij-chatgpt#27]
When the idea opens the chatgpt page box, it should take over the Enter shortcut key for quick interaction

**Comment by LiLittleCat** [comment]
@fwnan Are you suggesting adding a shortcut for opening the tool window?

**Comment by fwnan** [comment]
@LiLittleCat  No, I suggest pressing the Enter key to transport the issue like a web page, rather than manually clicking the send button now

**Comment by fwnan** [comment]
@LiLittleCat  No, I suggest pressing the Enter key to send the answer like a web page, rather than manually clicking the send button now.

**Comment by wq287627997** [comment]
I think so too,This would be very convenient.

**Comment by LiLittleCat** [comment]
Hello @fwnan, @wq287627997,
I've just upgraded to IntelliJ IDEA 2023.2.5 (Ultimate Edition, Build #IU-232.10227.8, released on November 9, 2023). Excitingly, I discovered that the 'Enter' shortcut is now supported by JCEF in this version. Notably, when typing in the input box, the icon at the end lights up, indicating that pressing Enter will send the message. I highly recommend updating to this latest version to see if it addresses your concerns as well.

![image](https://github.com/LiLittleCat/intellij-chatgpt/assets/40536573/1581e75d-8f0a-4bfb-9dd9-e5aa297aae3f)

![image](https://github.com/LiLittleCat/intellij-chatgpt/assets/40536573/063e4150-ef7a-44ac-9e9b-7bec49feaf5d)

Additionally, if you're unable to open the plugin in your current version, please refer to https://github.com/LiLittleCat/intellij-chatgpt/issues/29#issuecomment-1824067404 for assistance.

**Comment by MooMinIL** [comment]
**SOLVED! Edit below!**
Just wanted to update i upgraded to PyCharm 2023.3.1 (Community Edition) Build #PC-233.11799.298, built on December 12, 2023 and the Enter key to send prompts still doesn't work 😔

Edit: OK! so after playing around with ChatGPT in an external browser, i noticed that i had the same issue of the Enter key not sending my query when i shrunk down the browser window to a point where the view became only the conversation without the menu on the left (the one showing a list of old conversations)!
This means that if you extend the Chat GPT window of the plugin in the IDE to a point where the left menu pops up in the window, you can click enter directly to send a query.

This means the bug is on the ChatGPT interface side and not the plugin side.
